Server/ReasnAPI/ReasnAPI/Services/ParameterService.cs

```diff
@@ -1,16 +1,98 @@
 ﻿using ReasnAPI.Models.Database;
+using ReasnAPI.Models.DTOs;
+using System.Linq.Expressions;
 
-namespace ReasnAPI.Services {
-    public class ParameterService (ReasnContext context){
-        private readonly ReasnContext _context = context;
-
-        /* TODO: Create following functions for this class
-         * create
-         * update
-         * delete
-         * get by ID
-         * get list by filter
-         * get all
-         */
+namespace ReasnAPI.Services;
+public class ParameterService(ReasnContext context)
+{
+    public ParameterDto CreateParameter(ParameterDto parameterDto)
+    {
+        var parameter = context.Parameters.FirstOrDefault(r => r.Key == parameterDto.Key);
+        if (parameter != null && parameter.Value == parameterDto.Value)
```

Comment:
Wouldn't be easier to make it .FirstOrDefault(r => r.Key == parameterDto.Key && parameter.Value == parameterDto.Value)?

Reply:
Updated